### PR TITLE
Add Inscription and Transaction resources for oneclick mall

### PR DESCRIPTION
**Pull request by iescaida**
Added Oneclick mall Inscription methods start, finish and delete also Transaction methods authorize, refund and status

### CODE REVIEW

File: transbank/oneclick/transaction_mall.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import requests
```
Comment (goncafa) [inline transbank/oneclick/transaction_mall.py]: Name of the file should be mall_transaction.py

File: transbank/oneclick/transaction_mall.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+from transbank.error.transaction_status_error import TransactionStatusError
+from transbank.oneclick.request import TransactionAuthorizeRequest, TransactionRefundRequest, \
+    MallTransactionAuthorizeDetails
+from transbank.oneclick.response import TransactionAuthorizeResponse, TransactionRefundResponse, \
+    TransactionStatusResponse
+from transbank.oneclick.schema import TransactionAuthorizeRequestSchema, TransactionAuthorizeResponseSchema, \
+    TransactionRefundRequestSchema, TransactionRefundResponseSchema, TransactionStatusResponseSchema
+
+
+class Transaction(object):
```
Comment (goncafa) [inline transbank/oneclick/transaction_mall.py]: The class name should be MallTransaction

File: transbank/common/schema.py [hunk 3]
```diff
@@ -4,3 +4,9 @@
 class CardDetailSchema(Schema):
     card_number = fields.Str()
 
+
+class MallDetailsSchema(Schema):
+    commerce_code = fields.Str()
+    buy_order = fields.Str()
+    installments_number = fields.Int()
+    amount = fields.Float()
```
[the remark this answers is not in the document]
Comment (iescaida) [reply transbank/common/schema.py]: you were right, I thought other mall products could have used it, but this one has an extra property, cheers!

File: transbank/oneclick/__init__.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+from transbank.common.integration_type import IntegrationType
+
+commerce_code = "597055555541"  # Default COMMERCE_CODE for Patpass By Webpay
+api_key = "579B532A7440BB0C9079DED94D31EA1615BACEB56610332264630D42D0A36B1C"  # Default API_KEY for Patpass By Webpay
```
[the remark this answers is not in the document]
Comment (iescaida) [reply transbank/oneclick/__init__.py]: no they're not , copy/pate and forgot to change those comments, so lucky we have you to point this out

File: transbank/oneclick/inscription_mall.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    def delete(cls, token: str, user_name: str, options: Options = None):
+        options = cls.build_options(options)
+        endpoint = '{}/{}'.format(cls.__base_url(options.integration_type), 'inscriptions')
+
+        request = InscriptionDeleteRequest(user_name, token)
+        try:
+            requests.delete(url=endpoint, data=InscriptionDeleteRequestSchema().dumps(request).data,
+                            headers=HeadersBuilder.build(options))
+        except:
+            raise InscriptionDeleteError()
```
[the remark this answers is not in the document]
Comment (iescaida) [reply transbank/oneclick/inscription_mall.py]: done

File: transbank/oneclick/transaction_mall.py (hunk 2 above)
Comment (iescaida) [reply transbank/oneclick/transaction_mall.py]: done

File: transbank/oneclick/transaction_mall.py (hunk 1 above)
Comment (iescaida) [reply transbank/oneclick/transaction_mall.py]: done

File: transbank/oneclick/inscription_mall.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (goncafa) [reply transbank/oneclick/inscription_mall.py]: 🎉

**Pull request merged**